geode-core/src/main/java/org/apache/geode/distributed/internal/membership/InternalDistributedMember.java

```diff
@@ -549,7 +550,7 @@ public void setVersionObjectForTest(Version v) {
   }
 
   @Override
-  public Version getVersionObject() {
+  public VersionOrdinal getVersionObject() {
```

Comment:
This should continue to return a Version, not a VersionOrdinal.  We already have a method for getting the ordinal.

Reply:
Since `IDM` isa `MemberIdentifier` this method has to return a `VersionOrdinal` not a `Version` since, in general, a `MemberIdentifier` can be for a member that is running an unknown software version.

I'm renaming `MemberIdentifier.getVersionObject()` => `MemberIdentifier.getVersionOrdinalObject()` and adding a method `IDM.getVersionObject()` to really return the version for callers that really need it.

What's odd about the latter is that I don't see how that method could/should ever return anything but `Version.getCurrentVersion()` since the version, in this case, should be the version of the software running in the local JVM—which is always `Version.getCurrentVersion()`.